### PR TITLE
Done the third challenge, waiting for approve and ready to merge

### PR DESCRIPTION
**Pull request by Lye-Frozen-Heart**

### CODE REVIEW

File: challenge_03_loops-conditionals/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/README.md:1]: ⚠️ FIX: move `READEM.md` at project root level

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    3. Create an if statement that checks if animal is equal to "cat"
+    4. If it is equal to "cat", print out "Meow!" to the screen
+    5. If it is equal to "dog", print out "Woof!" to the screen
+    6. If it is neither "cat" nor "dog", print out "Must be an alien"
+    7. At the end of the program print out: "--- Animal check complete ---"
+    8. Change animal to different values to make sure it works in all cases
+*/
+console.log("--- Animal Checker Program ---");
+let animal = "alien";
+if (animal == "cat") console.log("Meow!");
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js:13]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    Create a program that starts with a number variable.
+    The program will return the variable is even or odd.
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+let numberToCheck = 14;
+if (numberToCheck % 2 == 0)
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js:11]: Even it is not necessary, it very recommended to use brackets when using if/else statement

```suggestion
if (...){
  //...
} else {
  //...
}
```

⚠️ FIX: format `if/else` statement

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+for (let i = 10; i <= 40; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js:6]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  if (i % 2 == 0) console.log("Pair: ", i);
+}
+for (let i = 10; i <= 40; i++) {
+  if (i % 2 == 1) console.log("Odd: ", i);
+}
+/**
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js:12]: A better solution...

```js
for (let i = 10; i <= 40; i++) {
  if (i % 2 === 0) console.log("Pair: ", i);
  if (i % 2 !== 0) console.log("Odd: ", i);
}
```

Take care of comparision operators. It's not the same using the equality operator (`==`) than usinng the strict equality operator (`===`).
Strict equality operator i sthe most recommende for most of the cases we will manage.

* [equality operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality)
* [strict equality operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_01.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+/*
+    1. Create a for loop that loops while index is less or equal than 10.
+    2. Each loop:
+        * create a new random number between 1 and 10 (hint: check documentation regarding Math.floor and Math.random methods)
+        * evaluate if random number is greater than 5
+        * if it is equal or greater than 5 print "Logged In!"
+        * if it is less than 5 print "Logged Out!"
+*/
+for (let i = 0; i <= 10; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_01.js:9]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    Checkpoint! 40
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+for (let i = 1; i < 100; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js:21]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_00.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    1. Create a variable called "loggedIn" that starts off as false.
+    2. Create a while loop that loops while "loggedIn" is false.
+    3. Each loop, print out: "Incorrect login credentials"
+    3. After 3 loop iterations, change "loggedIn" to be true.
+    4. After the loops is done print out: "Sucessfully logged in!"
+
+    HINT1: Be careful for an infinite loop!
+*/
+let loggedIn = false;
+let i = 0;
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_00.js:11]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+/*
+let incremental = 10;
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js:7]: ⚠️ FIX: remove commented code

**Review by tictools — CHANGES_REQUESTED** [review]

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js (hunk 4 above)
Comment (Lye-Frozen-Heart) [reply challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js:12]: Yeah I know it is not the same. The strict one also checks if the operands are from the same type, so I ensured that the checked number is always a number not a string or any other type. Even so for this basic exercise maybe it's not necessary.

**Review by nilete22 — APPROVED** [review]

**Pull request merged**